scripts/add_renewal_events.py

```diff
@@ -0,0 +1,46 @@
+import sys
+import uuid
+
+import os
+
+root_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.insert(0, root_dir)
+os.environ["DJANGO_SETTINGS_MODULE"] = "atmosphere.settings"
+
+import django
+
+django.setup()
+from jetstream.models import TASAPIDriver
+from core.models import EventTable, AllocationSource
+
+# fetch values from TACC
+api = TASAPIDriver()
+allocations = api.get_all_projects()
+
+for source in allocations:
+    for details in source['allocations']:
+
+        hashstring = '%s_%s' % (source['chargeCode'], details['id'])
+        hashed_uuid = uuid.uuid5(uuid.NAMESPACE_DNS, str(hashstring))
```

Comment:
Should this be `uuid.NAMESPACE_X500`? 

If so, I think the `hashstring`'s format would need to change - no?